website/client/src/components/achievements/levelUp.vue

```diff
@@ -1,109 +1,197 @@
 <template>
   <b-modal
     id="level-up"
-    :title="$t('levelUpShare')"
     size="sm"
-    :hide-footer="true"
-    :hide-header="true"
+    :title="title"
+    ok-only
+    :ok-title="$t('onwards')"
+    v-bind:footer-class="{ greyed: displayRewardQuest }"
   >
-    <div class="modal-body text-center">
-      <h2>{{ $t('reachedLevel', {level: user.stats.lvl}) }}</h2>
+    <section class="flex">
+      <span
+        class="star-group mirror"
+        v-html="starGroup"
+      ></span>
       <avatar
         class="avatar"
         :member="user"
       />
-      <p class="text">
-        {{ $t('levelup') }}
-      </p>
-      <button
-        class="btn btn-primary"
-        @click="close()"
-      >
-        {{ $t('onwards') }}
-      </button>
-      <br>
-      <!-- @TODO: Keep this? .checkboxinput(type='checkbox', v-model=
+      <span
+        class="star-group"
+        v-html="starGroup"
+      ></span>
+    </section>
+
+    <hr>
+    <p class="text">
+      {{ $t('levelup') }}
+    </p>
+
+    <section
+      v-if="displayRewardQuest"
+      class="greyed"
+    >
+      <div class="your-rewards flex">
+        <span
+          class="sparkles"
+          v-html="sparkles"
+        ></span>
+        <span class="text">{{ $t('yourRewards') }}</span>
+        <span
+          class="sparkles mirror"
+          v-html="sparkles"
+        ></span>
+      </div>
+
+      <div :class="questClass"></div>
+      <hr>
+    </section>
+    <!-- @TODO: Keep this? .checkboxinput(type='checkbox', v-model=
 'user.preferences.suppressModals.levelUp', @change='changeLevelupSuppress()')
 label(style='display:inline-block') {{ $t('dontShowAgain') }}
       -->
-    </div>
   </b-modal>
 </template>
 
 <style lang="scss">
   #level-up {
-    h2 {
-      color: #4f2a93;
+    @media (min-width: 576px) {
+      .modal-sm {
+        max-width: 330px;
+      }
+    }
+
+    header {
+      padding: 0;
+      border: none;
+
+      h5 {
+        margin: 31px auto 16px auto;
+        color: #4f2a93;
+      }
+
+      button {
+        position: absolute;
+        right: 18px;
+        top: 12px;
+      }
+    }
+
+    section {
+      width: 100%;
+      margin: 0;
+    }
+
+    hr {
+      margin: 0;
+      border-top: 25px solid rgb(255 63 249 / .2);
+    }
+
+    footer {
+      padding: 0;
+      border: none;
+
+      button {
+        margin: 0 auto 32px auto;
+      }
+    }
+
+    .flex {
```

Comment:
for this we can use the already existing `d-flex` class provided by bootstrap

Reply:
Done.